tools/CoqMakefile.in

```diff
@@ -162,6 +161,31 @@ ifdef DSTROOT
 DESTDIR := $(DSTROOT)
 endif
 
+# findlib files installation
+ifeq ($(DESTDIR),)
+FINDLIBPREINST= true
+FINDLIBDESTDIR= -destdir $(COQCORELIB)../
+else
+FINDLIBPREINST= mkdir -p $(DESTDIR)/$(COQCORELIB)../
+FINDLIBDESTDIR= -destdir $(DESTDIR)/$(COQCORELIB)../
```

Comment:
```suggestion
FINDLIBDESTDIR= -destdir $(COQCORELIB)/../
else
FINDLIBPREINST= mkdir -p $(DESTDIR)/$(COQCORELIB)/../
FINDLIBDESTDIR= -destdir $(DESTDIR)/$(COQCORELIB)/../
```
I think COQCORELIB is guaranteed to end with a / by coming from coq_makefile but this is nicer to read and matches other uses in this file.